_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,53 @@
+
+We made a heavily menu based game that would have been easy for us in React, or Elm, but in Unity we were blind. After a good amount of fighting with C#, we did end up with a couple working pages, but I expect there is a much easier way to build UI. C# isn't to blame for that, but we unanimously enjoy the languages that we write usually, but the support of Unity makes C# worth it.
+
+
+## Design
+
+Art takes so much longer than you expect it to! We worked on the project for about 15 hours, and it's crazy how few assets came out of that. In retrospect, there is also so much more to the aesthetic of your game than just the few images of game pieces. The designer's role can encompass the entire feel of the game. With [The Ferryman]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %}) we had a lot of text, menus, and backgrounds.
+
+Choosing a color pallet and letting Sam and RJ get at it just wasn't enough to give the in-game Bar the feeling that we wanted. Next time around, I think the design role will be a more holistic job, focused on putting less effort into more aspects of the game, instead of lots of effort into a handful of assets.
+
+
+## Timeline
+
+This part was awesome for us. Historically, we have had very soft deadlines, if any at all. The deadline was non-negotiable and it was short, we knew exactly how much effort we would be able to put in before losing our sanity. We prioritized ruthlessly, cutting out many "fun", but inessential features. At the end of Saturday morning, we had a plan for the minimal features that we could still consider a "game."
+
+Of course this didn't lead to the worlds most exciting game, but that wasn't our goal. We wanted to challenge ourself to plan well, and execute. That has been a problem in the past for us, and this was the best opportunity we've had to really test out or self-control.
+
+
+## So, what happened at the end?
+
+Sunday afternoon, wrapping up the game, trying to build a WebGL version to publish on the website and we realized we didn't read the rules very closely… **An additional rule farther down the page said, "no text".** Obviously that is a huge rule, and seeing as our game was basically all text, we couldn't submit. Despite how disqualifying our mistake was, we were on a mission to challenge ourselves, and we absolutely crushed it.
+
+We loved the jam; **Pushed ourselves to the limit of our sanity and physical well being**, learned an unearthly amount for a single weekend, and had a blast. To everyone out there that is thinking about making games. I was scared to do a game jam. We would go to events and everyone would suggest doing a jam, constantly telling us how good it would be for our personal development. They were right. We certainly didn't need to wait as long as we did. The great part is that signing up takes like 2 minutes, so if you can overcome your fear for that long, we will be able to do it!
```

Comment:
```suggestion
We loved the jam. **We pushed ourselves to the limit of our sanity and physical well being,** learned an unearthly amount for a single weekend, and had a blast. To everyone out there that is thinking about making games. I was scared to do a game jam. We would go to events and everyone would suggest doing a jam, constantly telling us how good it would be for our personal development. They were right. We certainly didn't need to wait as long as we did. The great part is that signing up takes like 2 minutes, so if you can overcome your fear for that long, you'll be able to do it!
```

Reply:
Bad diff tool... basically, "jam; **Pushed" -> "jam. **We pushed" and "we will be able to do it" -> "you'll be able to do it"

Reply:
TY, needed the explanation, diff too hard